src/Shared/Components/CICDHistory/DeploymentHistoryConfigDiff/DeploymentHistoryConfigDiffCompare.tsx

```diff
@@ -0,0 +1,123 @@
+import { useEffect } from 'react'
+import { generatePath, useRouteMatch } from 'react-router-dom'
+
+import { DeploymentConfigDiff, DeploymentConfigDiffProps } from '@Shared/Components/DeploymentConfigDiff'
+import { SortingOrder } from '@Common/Constants'
+import { useUrlFilters } from '@Common/Hooks'
+import {
+    getSelectPickerOptionByValue,
+    SelectPickerOptionType,
+    SelectPickerVariantType,
+} from '@Shared/Components/SelectPicker'
+import { ComponentSizeType } from '@Shared/constants'
+
+import { DeploymentHistoryDiffDetailedProps, DeploymentHistoryConfigDiffQueryParams } from './types'
+import { getPipelineDeploymentsOptions, parseDeploymentHistoryDiffSearchParams } from './utils'
+
+export const DeploymentHistoryConfigDiffCompare = ({
+    envName,
+    setFullScreenView,
+    pipelineDeployments,
+    wfrId,
+    previousWfrId,
+    convertVariables,
+    setConvertVariables,
+    runSource,
+    renderRunSource,
+    resourceId,
+    ...props
+}: DeploymentHistoryDiffDetailedProps) => {
+    // HOOKS
+    const { path, params } = useRouteMatch()
+
+    // URL FILTERS
+    const { compareWfrId, updateSearchParams, sortBy, sortOrder, handleSorting } = useUrlFilters<
+        'sort-config' | '',
```

Comment:
check if enum should be used. saw this at multiple places

Reply:
only one place